packages/fxa-auth-server/docs/swagger/account-docs.ts

```diff
@@ -0,0 +1,247 @@
+
+      This request will fail unless the account's email address and current session has been verified.
+    `,
+  ],
+  plugins: {
+    'hapi-swagger': {
+      responses: {
+        400: {
+          description: dedent`
+            Failing requests may be caused by the following errors (this is not an exhaustive list):
+            \`errno: 104\` - Unverified account
+          `,
+        },
+      },
+    },
+  },
+  tags: TAGS.ACCOUNT,
+};
+
+const ACCOUNT_UNLOCK_RESEND_CODE_POST = {
+  description: 'deprecated',
+  notes: ['This endpoint is deprecated.'],
+  plugins: {
+    'hapi-swagger': {
+      deprecated: true,
+      responses: {
+        410: {
+          description: dedent`
+            Failing requests may be caused by the following errors (this is not an exhaustive list):
+            \`errno: 116\` - This endpoint is no longer supported
+          `,
+        },
+      },
+    },
+  },
+  tags: TAGS.ACCOUNT,
+};
+
+const ACCOUNT_UNLOCK_VERIFY_CODE_POST = {
+  description: 'deprecated',
+  notes: ['This endpoint is deprecated.'],
+  plugins: {
+    'hapi-swagger': {
+      deprecated: true,
+      responses: {
+        410: {
+          description: dedent`
+            Failing requests may be caused by the following errors (this is not an exhaustive list):
+            \`errno: 116\` - This endpoint is no longer supported
+          `,
+        },
+      },
+    },
+  },
+  tags: TAGS.ACCOUNT,
+};
+
+const ACCOUNT_RESET_POST = {
+  description: '🔒 accountResetToken',
+  notes: [
+    dedent`
+      🔒 HAWK-authenticated with account reset token
+
+      This sets the account password and resets \`wrapKb\` to a new random value.
+
+      Account reset tokens are single-use and consumed regardless of whether the request succeeds or fails. They are returned by the \`POST /password/forgot/verify_code\` endpoint.
+
+      The caller can optionally request a new \`sessionToken\` and \`keyFetchToken\`.
+    `,
+  ],
+  plugins: {
+    'hapi-swagger': {
+      responses: {
+        400: {
+          description: dedent`
+            Failing requests may be caused by the following errors (this is not an exhaustive list):
+            \`errno: 108\` - Missing parameter in request body
+          `,
+        },
+      },
+    },
+  },
+  tags: TAGS.ACCOUNT,
+};
+
+const ACCOUNT_DESTROY_POST = {
+  description: '🔒🔓 sessionToken',
+  notes: [
+    dedent`
+      🔒🔓 Optionally HAWK-authenticated with session token
+
+      Deletes an account. All stored data is erased. The client should seek user confirmation first. The client should erase data stored on any attached services before deleting the user's account data.
+    `,
+  ],
+  plugins: {
+    'hapi-swagger': {
+      responses: {
+        400: {
+          description: dedent`
+            Failing requests may be caused by the following errors (this is not an exhaustive list):
+            \`errno: 103\` - Incorrect password
+            \`errno: 138\` - Unverified sessio
```

Comment:
```suggestion
            \`errno: 138\` - Unverified session
```

Reply:
@pdehaan with the sharp eyes! 😎 I'll be sure to update this, thanks!!